app/assets/stylesheets/application.scss

```diff
@@ -26,6 +26,7 @@
 @import "layout/footer";
 @import "components/masthead";
 @import "components/general";
+@import "components/userinfo_box";
```

Comment:
since 'user info' is actually two words, I think this file should actually be `_user_info_box`

Reply:
Changed name to _user_info_box as recommended